src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Models/FileUpload.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+
+namespace Kaiyuanshe.OpenHackathon.Server.Models
+{
+    /// <summary>
+    /// FileUpload
+    /// </summary>
+    public class FileUpload
+    {
+        /// <summary>
+        /// SAS token expiration time. 2 by default.
+        /// </summary>
+        /// <example>3</example>
+        [Range(2, 30)]
+        public int? expiration { get; set; }
+
+        /// <summary>
+        /// The name of the file to upload to the blob.
+        /// </summary>
+        /// <example>avatar.jpg</example>
+        [MaxLength(256)]
+        public string filename { get; set; }

```

Comment:
`required` or not? if not, we need to give it a default name .

Reply:
the filename is required